osu.Game/Overlays/Changelog/ChangelogBuild.cs

```diff
@@ -83,9 +83,9 @@ private void load(OsuColour colours)
                             {
                                 Anchor = Anchor.CentreLeft,
                                 Origin = Anchor.CentreRight,
-                                Size = new Vector2(fontSmall.Size),
+                                Size = new Vector2(10),
                                 Icon = entry.Type == ChangelogEntryType.Fix ? FontAwesome.Solid.Check : FontAwesome.Solid.Plus,
-                                Colour = entryColour,
+                                Colour = entryColour.Darken(0.7f),
```

Comment:
If we're going for 100% parity, this should be
```suggestion
                                Colour = entryColour.Opacity(0.5f),
```
![image](https://user-images.githubusercontent.com/20418176/75099249-abf07b00-55bf-11ea-8ded-e038853c54c1.png)